tests/network/curl_client.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE"s openQA tests
+# #
+# # Copyright © 2019 SUSE LLC
+# #
+# # Copying and distribution of this file, with or without modification,
+# # are permitted in any medium without royalty provided the copyright
+# # notice and this notice are preserved.  This file is offered as-is,
+# # without any warranty.
+#
+# # Summary: test that uses some aaa_base binaries command line tool to regression test.
+# # If succeed, the test passes without error.
+#
+# Summary: Test regression to curl.
+#  this test curl with http, https, ldap, ftp and ntlm auth
+#  poo#51536
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use mm_network 'setup_static_mm_network';
+use mm_tests;
```

Comment:
not needed too, not used anywehere

Reply:
ok, done! Thanks

Reply:
sorry I meant `mm_tests` not sure if you removed anything else. You don't use anything from it.